src/bootstrap/bootstrap.py

```diff
@@ -916,10 +916,8 @@ def build_bootstrap_cmd(self, env):
         env.setdefault("RUSTFLAGS", "")
         # we need to explicitly add +xgot here so that we can successfully bootstrap
         # a usable stage1 compiler
-        # FIXME: remove this if condition on the next bootstrap bump
-        # cfg(bootstrap)
-        if self.build_triple().startswith('mips'):
-            env["RUSTFLAGS"] += " -Ctarget-feature=+xgot"
+        env["RUSTFLAGS"] += " -Ctarget-feature=+xgot"
+
```

Comment:
https://github.com/rust-lang/rust/pull/113637

Reply:
just to be sure; does that `FIXME` means applying the flag unconditionally or not setting it at all anymore ?

cc @jyn514

Reply:
this is part of the target-spec now, so you can remove it altogether: https://github.com/rust-lang/rust/blob/bacf5bcbc7dae526872cf6af3fc0064a51b1efa2/compiler/rustc_target/src/spec/mips64_unknown_linux_gnuabi64.rs#L15
```suggestion
```